tests/unit/src/test_conditionsStore.cpp

```diff
@@ -218,46 +186,20 @@ SCENARIO( "Creating a ConditionsStore", "[ConditionsStore][Creation]" )
 			{
 				REQUIRE( store.Get("hello world") == 100 );
 				REQUIRE( store.Get("goodbye world") == 404 );
-				REQUIRE( primarySize(store) == 2 );
-				// Also check for possible ill-effects from primarySize() itself.
-				REQUIRE( primarySize(store) == 2 );
+				REQUIRE( store.PrimariesSize() == 2 );
+				// Also check for possible ill-effects from primariesSize() itself.
```

Comment:
```suggestion
				// Also check for possible ill-effects from PrimariesSize() itself.
```
Though, if `ConditionsStore::PrimariesSize()` is const-qualified, it shouldn't be changing anything about the store, size or anything else. And, in order to truly check this, we would need to check each condition individually and ensure that the values are unchanged, not just recheck the size.